.gitignore

```diff
@@ -21,3 +21,4 @@ test_uwsgi_failed
 .idea
 .pytest_cache/
 venv/
+.vscode
```

Comment:
Is this intentional?

Reply:
Yes but if you insist I will stop using VSCode and switch to Pycharm.